doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1078,6 +1078,7 @@ ExtensionArray
 - Fixed bug where :meth:`StringArray.memory_usage` was not implemented (:issue:`33963`)
 - Fixed bug where :meth:`DataFrameGroupBy` would ignore the ``min_count`` argument for aggregations on nullable boolean dtypes (:issue:`34051`)
 - Fixed bug that `DataFrame(columns=.., dtype='string')` would fail (:issue:`27953`, :issue:`33623`)
+- Bug where :class:`Series` set to scalar extension type was considered an object type rather than the extension type (:issue:`34832`)
```

Comment:
I find this sentence a bit hard to understand ("Series set to scalar extension type" -> it is not a Series directly that we are setting, but actually a column in a dataframe). Can you try to rephrase?

Reply:
For creating a Series directly this actually also fails on master:

```
In [14]: pd.Series(pd.Period("2012", freq="A"), index=[0, 1])   
Out[14]: 
0    2012
1    2012
dtype: object
```

and I assume this is not fixed by this PR (but fixing that is for a separate PR! ;-))

Reply:
Good point! Updated the wording